deploy.py

```diff
@@ -40,20 +40,21 @@ def open(name, *a, **kw):
         diff.update({b: False for b in remove})
     else:
         diff = {f: os.path.isfile(f) for f in diff}
-    deploy_data = {'subreddit': os.getenv('subreddit')}
-    for image, exists in diff.items():
-        data = dict(deploy_data)
-        with open(image) as image_file:
-            if exists:
-                data['image_path'] = image
-            else:
-                data['name'] = os.path.splitext(
-                    os.path.basename(image_file.name)
-                )[0]
-            getattr(
-                reddit,
-                "{0}_image".format("upload" if exists else "delete")
-            )(**data)
+    for subreddit in os.getenv('subreddit').split('+'):
+        deploy_data = {'subreddit': subreddit}
+        for image, exists in diff.items():
```

Comment:
I'm no saint on this, but any updates? Or if you don't have the time, may I squash and merge?